sources/packages/web/src/types/contracts/institution/InstitutionUser.ts

```diff
@@ -29,7 +29,7 @@ export interface InstitutionUserRoleLocation {
   locationId?: number;
   userType?: string;
 }
-export interface InstitutionUserDto {
+export interface InstitutionUserAPIOutDTO {
```

Comment:
this should be a class and `APIOut/InDTO` file name should end with `dto.ts`

Reply:
may be u can move it to a new `dto.ts` file

Reply:
Actually I discovered it has not been used anymore. I'm removing this one. Thanks!